dom/src/main/scala/org/scalajs/dom/DataTransferItemList.scala

```diff
@@ -0,0 +1,41 @@
+/** All documentation for facades is thanks to Mozilla Contributors at https://developer.mozilla.org/en-US/docs/Web/API
+  * and available under the Creative Commons Attribution-ShareAlike v2.5 or later.
+  * http://creativecommons.org/licenses/by-sa/2.5/
+  *
+  * Everything else is under the MIT License http://opensource.org/licenses/MIT
+  */
+package org.scalajs.dom
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+
+/** A list of [[DataTransferItem]] objects representing items being dragged. During a drag operation, each DragEvent has
+  * a dataTransfer property and that property is a DataTransferItemList.
+  */
+@js.native
+@JSGlobal
+class DataTransferItemList extends js.Object {
```

Comment:
Since the constructor cannot be used publicly.

```suggestion
class DataTransferItemList private[this] extends js.Object {
```